data-migration/migrate-sharding-mysql-tidb-less-tb.md

```diff
@@ -0,0 +1,241 @@
+ignore-checking-items: ["auto_increment_ID"]  # In this example, there are auto-incremental primary keys upstream, so you do not need to check this item.
+
+target-database:
+  host: "${host}"                             # For example: 192.168.0.1
+  port: 4000
+  user: "root"
+  password: "${password}"                     # Plaintext passwords are supported but not recommended. It is recommended that you use dmctl encrypt to encrypt plaintext passwords.
+
+mysql-instances:
+  -
+    source-id: "mysql-01"                                    # ID of the data source. You can obtain it from the data source configurations.
+    route-rules: ["sale-route-rule"]                         # The table route rules applied to the data source
+    filter-rules: ["store-filter-rule", "sale-filter-rule"]  # Binlog event filter rules applied to the data source
+    block-allow-list:  "log-bak-ignored"                     # Block & Allow Lists rules applied to the data source
+  -
+    source-id: "mysql-02"
+    route-rules: ["sale-route-rule"]
+    filter-rules: ["store-filter-rule", "sale-filter-rule"]
+    block-allow-list:  "log-bak-ignored"
+
+# Configurations for merging MySQL shards
+routes:
+  sale-route-rule:
+    schema-pattern: "store_*"
+    table-pattern: "sale_*"
+    target-schema: "store"
+    target-table:  "sale"
+
+# Filters some DDL events.
+filters:
+  sale-filter-rule:
+    schema-pattern: "store_*"
+    table-pattern: "sale_*"
+    events: ["truncate table", "drop table", "delete"]
+    action: Ignore
+  store-filter-rule:
+    schema-pattern: "store_*"
+    events: ["drop database"]
+    action: Ignore
+
+# Block and allow list
+block-allow-list:
+  log-bak-ignored:
+    do-dbs: ["store_*"]
+```
+
+The above example is the minimum configuration to perform the migration task. For more information, see [DM Advanced Task Configuration File](https://docs.pingcap.com/tidb-data-migration/stable/task-configuration-file-full/).
+
+For more information on `routes`, `filters` and other configurations in the task file, see the following documents:
+
+- [Table routing](https://docs.pingcap.com/tidb-data-migration/stable/key-features#table-routing)
+- [Block & Allow Table Lists](https://docs.pingcap.com/tidb-data-migration/stable/key-features#block--allow-table-lists)
+- [Binlog event filter](https://docs.pingcap.com/tidb-data-migration/stable/key-features#binlog-event-filter)
+- [Binlog expression filter](https://docs.pingcap.com/tidb-data-migration/stable/feature-expression-filter)
+
+## Step 3: Start the task
+
+You can use the [Pre-check](https://docs.pingcap.com/tidb-data-migration/stable/precheck) feature to spot configuration errors in the data migration task in advance. When data migration starts, DM will automatically check related permissions and configurations.
+
+You can also use the `check-task` command to manually check whether the configurations of the upstream MySQL instance meet the requirements.
```

Comment:
```suggestion
Before starting a migration task, run the `check-task` command to check whether the configuration meets the requirements of DM so as to avoid possible errors.
```
根据原文做的修改